ui/README.md

```diff
@@ -11,6 +11,15 @@ build tool and [pnpm](https://pnpm.io/) as the package manager.
 
 The UI expects ORT Server to be running locally.
 
+## Run the UI
+
+Here are the instructions to start the UI in local development mode:
+
+1. Go to the `ui` folder.
+2. Run `pnpm install` followed by `pnpm dev`.
+3. Ctrl-click the shown `http://localhost:5173/` link.
```

Comment:
The URL in Markdown would be more idiomatic in <>.

```suggestion
3. Ctrl-click the shown <http://localhost:5173/> link.
```

Reply:
My intention was to quote the URL as it is printed to the console, i. e. resemble console output. That's why I used backticks (and also did not make the URL clickable as a link).